R/cromwellValidate.R

```diff
@@ -15,18 +15,18 @@ cromwell_validate <- function(
   check_url(url)
   crom_mssg("Validating a workflow for Cromwell")
 
-  body <- list(workflowSource = httr::upload_file(wdl))
+  body <- list(workflowSource = curl::form_file(wdl))
   if (!is.null(all_inputs)) {
     body <- c(
       body,
-      workflowInputs = list(httr::upload_file(all_inputs))
+      workflowInputs = list(curl::form_file(all_inputs))
     )
   }
 
-  http_post(
+  http_req_post(
     url = make_url(url, "api/womtool/v1/describe"),
-    body = body,
-    encode = "multipart",
     token = token
-  )
+  ) |>
+    req_body_multipart(!!!body) |>
```

Comment:
!!! Yo !!! Would httr2::req_body_file() help to avoid this by any chance? See: https://stackoverflow.com/questions/77193596/upload-file-with-method-post-with-httr2-in-r

Reply:
Can you explain. Is the idea to avoid `!!!`? Or are you just thinking `req_body_file` is more appropriate?

Reply:
I don't think `req_body_file` works here b/c that fxn only handles one file at a time, and we can have 1 or 2 files depending on the input

Reply:
@seankross you okay with this? I need an approval before merging, plz&thx

Reply:
Ah I see, never mind then.